network.sh

```diff
@@ -139,7 +139,9 @@ elif [[ "$IP_STACK" = "v6" ]]; then
   export SERVICE_SUBNET_V4=""
   export SERVICE_SUBNET_V6=${SERVICE_SUBNET_V6:-"fd02::/112"}
   export NETWORK_TYPE=${NETWORK_TYPE:-"OVNKubernetes"}
-  export MIRROR_IMAGES=true
```

Comment:
Instead of introducing a new variable could we just change this so that it only happens if unset? e.g. 
> export MIRROR_IMAGES=${MIRROR_IMAGES:-true}

Reply:
I changed it to do this. Also update all the conditions on `MIRROR_IMAGES` to treat value `false` the same way it treats `unset`.

Reply:
I changed it to this:
```
 # MIRROR_IMAGES -
-# Mirror latest ci images to local registry. This is always true for IPv6, but
-# can be turned off for an IPv4 install.
+# Mirror latest ci images to local registry. This is true by default for IPv6, but
+# can be turned on for an IPv4 install. To explicitly disable set it to false.
 # Default: true
 #
 #export MIRROR_IMAGES=true
```

NOTE the change `can be turned off for an IPv4` to `can be turned on for an IPv4`, I think that is more correct. It is by default left `unset` when `IP_STACK=v4`, so in that case one would have to explicitly enable it? The "Default: true" bit is still correct because the default for `IP_STACK` is `v6`.